streamtrace.cc

```diff
@@ -1865,6 +1865,19 @@ debug_trace_entry::debug_trace_entry(const debug_trace_entry_disk_v1 &d,
 	decode_entry(*this, d.version, d.val1, d.val2);
 }
 
+debug_trace_entry::debug_trace_entry() :
```

Comment:
Don't do this.  Making `debug_trace_entry` have an out-of-line non-trivial default constructor will result in a bunch of redundant work being done.  If you must have the fields zero-initialised, then provide inline initialisers in the class definition.